plugins/mumble_plugin_linux.h

```diff
@@ -78,12 +79,13 @@ static inline int8_t isProcessWin32(const procid_t &pid) {
 
 	const char *filename = basename(path);
 
-	free(path);
-
 	if (strcmp(filename, "wine-preloader") == 0 || strcmp(filename, "wine64-preloader") == 0) {
+		free(path);
 		return 1;
 	}
 
+	free(path);
```

Comment:
Why not freeing `path` before the if-statement? To it seems as if it had to be freed either way...

E.g.
```
free(path);

if (strcmp(filename, "wine-preloader") == 0 || strcmp(filename, "wine64-preloader") == 0) {
	return 1;
}
```

Reply:
From 7a40719dfa0c1fd60b69e2d41ed7a876e5955c58:
> basename() returns a pointer to the position where the basename starts in the string passed as argument.

Reply:
Ah okay then  :bulb:

Reply:
I'd add a comment about that then. Otherwise someone like me looking at that code might "optimize" it and break it

Reply:
I agree.

Reply:
Done in #4099.